applications/FluidDynamicsHydraulicsApplication/custom_utilities/hydraulic_fluid_auxiliary_utilities.cpp

```diff
@@ -0,0 +1,322 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:   Uxue Chasco
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "processes/find_global_nodal_neighbours_process.h"
+#include "includes/global_pointer_variables.h"
+#include "utilities/parallel_utilities.h"
+#include "utilities/reduction_utilities.h"
+#include "spatial_containers/bins_dynamic.h"
+#include "utilities/rbf_shape_functions_utility.h"
+#include "utilities/divide_triangle_3d_3.h"
+
+// Application includes
+#include "hydraulic_fluid_auxiliary_utilities.h"
+
+namespace Kratos
+{
+double HydraulicFluidAuxiliaryUtilities::CalculateWettedPetimeter
+(
```

Comment:
```suggestion
double HydraulicFluidAuxiliaryUtilities::CalculateWettedPetimeter(
```
Please follow the style guide.